src/foam/box/MessagePortBox.js

```diff
@@ -27,25 +27,33 @@ foam.CLASS({
     'foam.box.Message'
   ],
 
-  imports: [ 'messagePortService', 'me' ],
+  imports: [
+    'generator? as ctxGenerator',
```

Comment:
I'd be fine with a rename of "generator". I was thinking of the "parser/generator" jargon, but it is true that generator means something else in other CS contexts.

You have a preference? Serializer?